laalaa/apps/advisers/migrations/0009_adding_discrimination_and_education_categories.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-08-30 16:16
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import connection
+
+
+def create_discrimination_and_education_categories(apps, schema_editor):
+    Category = apps.get_model("advisers", "Category")
+    # Hard coding the id values as this is referenced in advisers/fixtures
+    Category.objects.get_or_create(id=14, code="DISC", defaults={"civil": True})
+    Category.objects.get_or_create(id=15, code="EDU", defaults={"civil": True})
+
+    # This is required because the id field is specified in the fixtures (and above) and has become out of sync with
+    # next sequence value
+    sql = 'SELECT setval(pg_get_serial_sequence(\'advisers_category\',\'id\'), coalesce(max("id"), 1), max("id") IS NOT null) FROM "advisers_category";'
```

Comment:
I think you can remove the `coalesce` part here, and set the third argument to "false" because you know the table is always populated at the time it will run.

i.e. the following should be equivalent:
`SELECT setval(pg_get_serial_sequence(\'advisers_category\',\'id\'), max("id"), false) FROM "advisers_category"`
or even 
`SELECT setval(pg_get_serial_sequence(\'advisers_category\',\'id\'), 15) FROM "advisers_category"`

which is a bit easier to understand from the context.

Reply:
That was the output from the `./manage sqlsequencereset` command that prints the SQL statements for resetting sequences for the given app name(s).

Reply:
Ah ok fair enough, didn't know you could autogenerate that!